libs/reports-list/src/lib/reports-list/reports-list.component.ts

```diff
@@ -10,7 +15,20 @@ export class ReportsListComponent implements OnInit {
   @Input()
   reportsList: VaccinesReport[];
 
-  constructor() {}
+  constructor(public dialog: MatDialog) {}
 
   ngOnInit(): void {}
+
+  openDialog(report: VaccinesReport): void {
+    const dialogRef = this.dialog.open(ReportModalComponent, {
+      width: '350px',
+      height: '230px',
+      data: report,
+      direction: 'rtl'
```

Comment:
How does it works with dates and number?
(I know it works, I saw your screenshot. I'm asking how).

Reply:
i don't understand the question.

Reply:
I know that setting direction to RTL causes to numbers to get swapped. However, in the screenshot you sent it works properly. I'm asking how's that.

Reply:
oh. i define to angular how to format the date:
{{ report.endTime | date:'IST':'il' }}
it formats the date according to israel's locale

Reply:
I'm not sure it's related, these are two different mechanisms. The angular formatting is a string manipulation, just to decide on formatting. The issue I'm talking about is related to the rendering mechanism and I'm expecting it to be agnostic to the date string - i.e. even simple number like 123 will be shown like 321 when rendered with rtl.